packages/cms-lib/http.js

```diff
@@ -9,7 +9,7 @@ const { logger } = require('./logger');
 const {
   FileSystemErrorContext,
   logFileSystemErrorInstance,
-} = require('./errorHandlers');
+} = require('./errorHandlers/fileSystemErrors');
```

Comment:
@drewjenkins this seemed like the simplest fix for now. Does this seem reasonable?